Makefile

```diff
@@ -97,14 +97,23 @@ clean: $(clean-targets)
 
 # Assemble Change log.
 changelog:
-	@if [[ -z "$(NEXT_VERSION)" ]]; then \
-		echo "Error: Could not compute project's new version."; \
-		exit 1; \
-	else \
-		echo "Generating changelog for version $(NEXT_VERSION)..."; \
-		towncrier build --version $(NEXT_VERSION); \
-		echo "Next, review the staged changes, commit them and make a pull request."; \
-    fi
+	@$(ENSURE_NEXT_VERSION)
+	@$(ECHO_STDERR) "Generating Change Log for version $(NEXT_VERSION)..."
+	towncrier build --version $(NEXT_VERSION)
+	@$(ECHO_STDERR) "Next, review the staged changes, commit them and make a pull request."
+
+# Tag the next release.
+tag-next-release:
+	@$(ENSURE_NEXT_VERSION)
+	@$(ECHO_STDERR) "Checking if we can tag version $(NEXT_VERSION) as the next release..."
+	@$(ENSURE_NO_CHANGELOG_FRAGMENTS)
```

Comment:
Ah probably it was due to this missing check :-)

Reply:
Yes, this check will ensure this can't happen in the future :slightly_smiling_face:.